pyoverkiz/models.py

```diff
@@ -185,8 +187,8 @@ def __init__(
         self.controllable_name = controllable_name
         self.states = States(states)
         self.data_properties = data_properties
-        self.widget = widget
-        self.ui_class = ui_class
+        self.widget = UIWidget(widget)
+        self.ui_class = UIClass(ui_class)
```

Comment:
We should probably add the missing methods to these enums in case Somfy add a new widget.

Reply:
Done